MetaLamp/nft-marketplace/src/Plutus/Abstract/Percentage.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+
+module Plutus.Abstract.Percentage where
+import qualified Data.Aeson   as J
+import           GHC.Generics (Generic)
+import qualified PlutusTx
+import           Prelude      hiding (Fractional)
+import qualified Schema
+
+type Fractional = (Integer, Integer)
```

Comment:
I'd recommend use record type instead of synonym. This type can be misleading on what kind values it should carry when we're dealing with tuples.